api.md

```diff
@@ -2989,28 +2983,40 @@ used to implement the policy succeeded and when it is scheduled to run next.
 
 |Name|Description|
 |---|---|
-| `hypertable` | (REGCLASS) The name of the hypertable on which the policy is applied |
-| `job_id` | (INTEGER) The id of the background job created to implement the policy |
-| `job_type` | (TEXT) The type of policy the job was created to implement |
-| `last_run_success` | (BOOLEAN) Whether the last run succeeded or failed |
-| `last_finish` | (TIMESTAMPTZ) The time the last run finished |
-| `last_start` | (TIMESTAMPTZ) The time the last run started |
-| `next_start` | (TIMESTAMPTZ) The time the next run will start |
-| `total_runs` | (INTEGER) The total number of runs of this job |
-| `total_failures` | (INTEGER) The total number of times this job failed |
+|`hypertable` | (REGCLASS) The name of the hypertable on which the policy is applied |
+|`job_id` | (INTEGER) The id of the background job created to implement the policy |
+|`last_run_started_at`| Start time of the last job|
+|`last_successful_finish`| Time when the job completed successfully|
+|`last_run_status` | Whether the last run succeeded or failed |
+|`job_status`| Status of the job . Valid values are ‘Running’ and ‘Scheduled’|
+|`last_run_duration`| Duration of last run of the job|
+|`next_scheduled_run` | Start time of the next run |
+|`total_runs` | The total number of runs of this job|
+|`total_successes` | The total number of times this job succeeded |
+|`total_failures` | The total number of times this job failed |
 
 #### Sample Usage [](timescaledb_information-policy_stats-examples)
 
-Get information about statistics on created policies.
-```sql
-SELECT * FROM timescaledb_information.policy_stats;
-
-       hypertable       | job_id |  job_type   | last_run_success |         last_finish          |          last_start          |          next_start          | total_runs | total_failures
-------------------------+--------+-------------+------------------+------------------------------+------------------------------+------------------------------+------------+----------------
- conditions             |   1001 | drop_chunks | t                | Fri Dec 31 16:00:01 1999 PST | Fri Dec 31 16:00:01 1999 PST | Fri Dec 31 16:00:02 1999 PST |          2 |              0
-(1 row)
 ```
-
+Get information about continuous aggregate policy related statistics
+``` sql
```

Comment:
Can we give a simpler example that doesn't involve a JOIN in addition?

Reply:
I added another example without a join.